tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py27,py34
+
+[testenv]
+commands = pytest tests
+
+[testenv:py27]
+deps =
+    pytest
+    Cython==0.29.24
+    Pillow==5.4.1
+    PyPDF2==1.25.1
+
+[testenv:py34]
+deps =
+    pytest
+    Cython==0.29.24
+    Pillow==5.4.1
+    PyPDF2==1.25.1
```

Comment:
AFAIR, if both environments have the exact same requirements you can just specify them in the `[testenv]` section.

Reply:
good